cinder-{{cookiecutter.driver_name_lc}}/src/reactive/cinder_{{cookiecutter.driver_name_lc}}_handlers.py

```diff
@@ -29,5 +29,5 @@
 
 @charms.reactive.when('config.changed.driver-source')
 def reinstall():
-    with charms_openstack.charm.provide_charm_instance() as charm:
+    with charms_openstack.charm.provide_charm_instance() as charm: # noqa
```

Comment:
Whilst it fixes the bug, I don't think it's the appropriate fix.  It's a sledgehammer to tap a nail :)

The issue is that "charm" above (on line 20) is being shadowed by "charm" here.  The appropriate fix is to rename the variable here to something other than 'charm'.

There is an even better fix, which is to not do the import of "charm.openstack.cinder_{{ ...}}" but instead to use the auto-discovery feature of `charms.openstack` instead of that import:

```python
import charms_openstack.bus
charms_openstack.bus.discover()
```

I'm also slightly suspicious of the `charm.openstack.cinder_{{...}}` name as it might be `charms.openstack...` -- might be worth checking.

Reply:
Okay! Thanks for the comments Alex. I have made the import change that you suggested and re-verified that pep8, unit tests, and functional tests are all still working. Stand by for a new PR!